MIGRATING-0.6-0.7.md

```diff
@@ -331,10 +365,24 @@ It now also disallows out of bounds drawing by default. This behaviour can be ch
 
 ## Style module
 
-The `style` module has been removed.
+The `style` module has been removed. The items in it have been moved:
 
-TODO: Improve this section before release.
+- `PrimitiveStyle` and `PrimitiveStyleBuilder` are now available under `embedded_graphics::primitives::{PrimitiveStyle, PrimitiveStyleBuilder}`.
+- `TextStyle` and `TextStyleBuilder` were renamed are now available under `embedded_graphics::mono_font::{MonoTextStyle, MonoTextStyleBuilder}`.
+
+  Note that usage with `Text` has changed. See [the text changes section](#Text-rendering) for more.
+
+## Text and fonts
+
+The collection of builtin fonts are now sourced from public domain BDF fonts in the XOrg project. Due to this, they have slightly different dimensions and glyphs and so have changed names. Some sizes are not the same in the new set, but a rough mapping is as follows:
+
+- `fonts::Font6x6` -> Removed
+- `fonts::Font6x8` -> `mono_font::[ascii|latin1]::Font6x10`. If a display performs better when using 8px high fonts, `mono_font::[ascii|latin1]::Font5x8` is suggested instead.
+- `fonts::Font6x12` -> `mono_font::[ascii|latin1]::Font6x12`
+- `fonts::Font8x16` -> `mono_font::[ascii|latin1]::Font8x13`
+- `fonts::Font12x16` -> `mono_font::[ascii|latin1]::Font9x15`
+- `fonts::Font24x32` -> `mono_font::[ascii|latin1]::Font10x20` (this is the largest font available, however the old `Font24x32` was just a scaling of `Font12x16`)
```

Comment:
I'm not sold on the note. While true, "just a scaling" doesn't help if someone needs a bigger font. This also means the replacement isn't a good one (as rfuest's picture shows). I think we should just state that `fonts::Font24x32` has been removed.